### DISCUSSION TITLE
Is it possible to call the Etsy API for revenue stats?

### DISCUSSION THREAD

**Discussion by GabrielMolesley** [etsy/open-api#832] [Q&A]
Hi everybody, i was wondering if there was a way to get the revenue stats via the Etsy API. Any help would be appreciated.
- Gabriel

**Comment by turboturtleD** [comment]
I think there are ledger endpoints you could look at.  You'd likely have to crunch the numbers yourself though.  I don't think there is much in the line of statistical data coming out of the developer interfaceable part of the v3 API.

**Comment by anastasiam** [comment]
Hello! I'm also intrested in getting stats data through API by date range. Like the one that you can see in Listing Stats page.
